Tests/BUILD

```diff
@@ -1,40 +1,30 @@
 load("@build_bazel_rules_swift//swift:swift.bzl", "swift_library")
-
 load(
     "@build_bazel_rules_apple//apple:ios.bzl",
+    "ios_application",
     "ios_unit_test",
-    "ios_application"
 )
-
 load(
     "@build_bazel_rules_apple//apple:macos.bzl",
-    "macos_unit_test"
-)
-
-load(
-    "@build_bazel_rules_apple//apple:tvos.bzl",
-    "tvos_unit_test",
-    "tvos_application"
+    "macos_unit_test",
 )
 
 MINIMUM_IOS_VERSION = "9.0"
 
 MINIMUM_MACOS_VERSION = "10.9"
 
-MINIMUM_TVOS_VERSION = "14.5"
-
 swift_library(
     name = "KronosTests",
     srcs = glob(["KronosTests/*.swift"]),
-    deps = ["//:Kronos"]
+    deps = ["//:Kronos"],
```

Comment:
I have `buildifier` via VSCode and it does not auto add these trailing commas or performs sorting. How did you set that up?

Reply:
hrm, I didn't change anything, maybe it's a newer version?